Sources/GutenbergKit/Sources/EditorViewController.swift

```diff
@@ -11,8 +11,9 @@ public final class EditorViewController: UIViewController, GutenbergEditorContro
     private let controller = GutenbergEditorController()
     private let timestampInit = CFAbsoluteTimeGetCurrent()
     private let service: EditorService
-    private let siteURL: String
+    private let siteAPIURL: String
```

Comment:
WDYT about `siteApiRoot`?

Reply:
Sure! I've updated it in https://github.com/wordpress-mobile/GutenbergKit/commit/f268246891ae3a76be68c72a409e5f5ee64d40c2#diff-8d684f85d6faf91b7cef2e39f371584ec64f48bee75eec04292272b85272a5d4